firefox-ios/Client/Assets/CC_Script/fathom.mjs

```diff
@@ -1634,7 +1634,7 @@ class InwardRhs {
     type(theType) {
         // In the future, we might also support providing a callback that receives
         // the fnode and returns a type. We couldn't reason based on these, but the
-        // use would be rather a consise way to to override part of what a previous
+        // use would be rather a consise way to override part of what a previous
```

Comment:
Thanks for spotting this, but this file is [a shared file from mozilla-central](https://searchfox.org/mozilla-central/source/toolkit/modules/third_party/fathom/fathom.mjs#2), so changing it here will not affect anything. I suggest we skip this change in this PR. 

If you'd like to change this, I suggest opening a patch against mozilla-central ( https://firefox-source-docs.mozilla.org/setup/contributing_code.html ) and following the steps here in the [readme](https://searchfox.org/mozilla-central/source/toolkit/modules/third_party/fathom/README). This will sync up and change the file here as well eventually.